Hoxtygen/refugee-tales-frontend
# #38370681: setup state management with context api

##  What does this PR do
Creates a state management store for the entire application

##  Description of the task to be completed
* create folder and files structure needed for the different states.


##  How should this manually be tested
* clone the app
* open your terminal and run `git clone <app URL>`
* cd into the project folder named client and run `npm install`
* then run `npm start`
* You should see data rendered to the UI coming from one of the contexts created.

##  What are the relevant board stories
[#38370681](https://github.com/Hoxtygen/refugee-tales-frontend/projects/1#card-38370681)
